### PR TITLE
Issue 742

### LINKED ISSUE #742: [cleanup] Remove unused flags and functions in fairscale/benchmarks/pipe.py
## Cleanup

fairscale/benchmarks/pipe.py has unused flags and functions that need to be cleaned up. The following is a non exhaustive list:
* Rename MPI_PORT to PG_PORT or something that represents what the constant is used for.
* Remove eval function since we don't benchmark it.
* Remove flags "hostname" and other unused flags.

### PR DESCRIPTION
**Pull request by SimranKoul7**
## What does this PR do?
Fixes #742 

## Before submitting

- [ ] Did you have fun?
  - Yes, it was a great experience.
- [ ] Did you read the [contributor guideline](https://github.com/facebookresearch/fairscale/blob/master/CONTRIBUTING.md)?
   -Yes
- [ ] Was this discussed/approved via a Github issue? (no need for typos, doc improvements)
  - [ ] N/A
- [ ] Did you make sure to update the docs?
  - [ ] N/A
- [ ] Did you write any new necessary tests?
  - [ ] N/A
- [ ] Did you update the [changelog](https://github.com/facebookresearch/fairscale/blob/master/CHANGELOG.md)? (if needed)
  - [ ] N/A

## PR review
Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

#Changes made:
   - Changing name of MPI_PORT to PG_PORT
   - Commented function eval
   - Removed unused hostname flag

### CODE REVIEW

**Comment by min-xu-ai** [comment]
Hi @SimranKoul7, any updates? No hurry, just pinging.

File: benchmarks/pipe.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 from fairscale.nn.model_parallel import initialize_model_parallel
 from fairscale.utils.testing import dist_init
 
-MPI_PORT = 29500
+PG_PORT = 29500
```
Comment (anj-s) [inline benchmarks/pipe.py:30]: Do we need RPC_PORT as well? Can we remove that if it is not needed?

File: benchmarks/pipe.py [hunk 2]
```diff
@@ -232,7 +232,7 @@ def get_batch(source):
     else:
         return 0.0, 0.0
 
-
+'''
 # TODO(anj-s): Add an option for users to be able to benchmark evaluate.
```
Comment (anj-s) [inline benchmarks/pipe.py:236]: Lets remove this function instead of commenting it out.

**Comment by anj-s** [comment]
Thank you for the PR! Can we update the PR description to something that more closely reflects the changes?

**Comment by min-xu-ai** [comment]
@anj-s do we still need this?

**Comment by anj-s** [comment]
> @anj-s do we still need this?

We do but there has been no response from the PR owner so I am closing it.

**Pull request closed without merging**